Can't install due to bad peer dependency

**Describe the bug**
Install fails as the peer dependency for class transformer fails. 

```bash
$ npm install discord-nestjs discord.js
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: bot@0.0.1
npm ERR! Found: class-transformer@0.4.0
npm ERR! node_modules/class-transformer
npm ERR!   peerOptional class-transformer@"*" from @nestjs/common@7.6.13
npm ERR!   node_modules/@nestjs/common
npm ERR!     @nestjs/common@"^7.6.13" from the root project
npm ERR!     peer @nestjs/common@"^7.6.11" from discord-nestjs@1.1.7
npm ERR!     node_modules/discord-nestjs
npm ERR!       discord-nestjs@"*" from the root project
npm ERR!     4 more (@nestjs/core, @nestjs/microservices, ...)
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer class-transformer@"4.0.0" from discord-nestjs@1.1.7
npm ERR! node_modules/discord-nestjs
npm ERR!   discord-nestjs@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See /home/steve/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/steve/.npm/_logs/2021-03-04T17_05_30_141Z-debug.log

```

**To Reproduce**
Steps to reproduce the behavior:
1. Run `npm install discord-nestjs discord.js`
2. See error

**Expected behavior**
Package installs successfully 

**Additional context**
It looks like a typo in the peer dependencies: it is requiring class transformer v4.0.0 whereas the latest release is v0.4.0

--- a/package.json
+++ b/package.json
@@ -71,7 +71,7 @@
   "peerDependencies": {
     "@nestjs/common": "^7.6.11",
     "@nestjs/core": "^7.6.11",
-    "class-transformer": "4.0.0",
+    "class-transformer": "0.4.0",
     "class-validator": "0.13.1",
     "discord.js": "^12.5.1",
     "reflect-metadata": "^0.1.13",

